psl-core/src/test/java/org/linqs/psl/evaluation/statistics/RankingEvaluatorTest.java

```diff
@@ -73,4 +158,56 @@ public void testNegativeAUPRC() {
             }
         }
     }
+
+    @Test
+    public void testMeanAveragePrecision() {
+        // TODO Need more tests
+        for (double threshold = 0.1; threshold <= 1.0; threshold += 0.1) {
+            RankingEvaluator computer = new RankingEvaluator(threshold);
+            computer.compute(trainingMap, predicate);
+            double value = computer.meanAveragePrecision();
+
+            if (threshold <= 0.8) {
+                assertEquals("Threshold: " + threshold, 1.0, value, MathUtils.EPSILON);
+            } else {
+                assertEquals("Threshold: " + threshold, 1.0, value, MathUtils.EPSILON);
+            }
+        }
+    }
+
+    @Test
+    public void testDiscountedCumulativeGain() {
+        // TODO Need more tests
+        for (double threshold = 0.1; threshold <= 1.0; threshold += 0.1) {
+            RankingEvaluator computer = new RankingEvaluator(threshold);
+            computer.compute(trainingMap, predicate);
+            double value = computer.discountedCumulativeGain();
+
+            if (threshold <= 0.8) {
+            } else {
+            }
+        }
+    }
+
+    @Test
+    public void testNormalizedDiscountedCumulativeGain() {
+        // TODO Need more tests
+        for (double threshold = 0.1; threshold <= 1.0; threshold += 0.1) {
+            RankingEvaluator computer = new RankingEvaluator(threshold);
+            computer.compute(trainingMap, predicate);
+            double value = computer.normalizedDiscountedCumulativeGain();
+
+            if (threshold <= 0.8) {
+                assertEquals("Threshold: " + threshold, 1.0, value, MathUtils.EPSILON);
+            } else {
+                assertEquals("Threshold: " + threshold, 1.0, value, MathUtils.EPSILON);
+            }
+        }
+    }
+
+    @Test
+    public void testGetAllStats() {
```

Comment:
I'm not a fan of this test.
It doesn't evaluate anything.

Coverage tests are a bit deceptive (and not super useful in a compiled language), so let's get rid of it.